R/package.R

```diff
@@ -87,6 +87,7 @@ initialize_python <- function(required_module = NULL, use_environment = NULL) {
                           Sys.getenv("PATH"),
                           sep = .Platform$path.sep))  
   
+  trace_ms <- Sys.getenv("RETICULATE_TRACE", 0)
```

Comment:
Let's name this a bit more specifically (since `TRACE` could refer to general diagnostic logging). How about `RETICULATE_DUMP_STACK_TRACE`  or something like that?

Reply:
changed